### PR TITLE
Update docs structure and config guides

### PR DESCRIPTION
**Pull request by adnanrahic**
This PR updates the docs structure and config guides.

## Changes

-

## Fixes

- [x] #1919 
- [x] #1982 

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

**Comment by xoscar** [comment]
@adnanrahic is there a good way to test the documentation locally? like running a command and looking directly at the website?

**Comment by adnanrahic** [comment]
@xoscar I run the build and then serve the build directory.
```
npm run build
npm run serve
```

This shows any errors and what the live version of the docs will look like.

File: docs/docs/configuration/connecting-to-data-stores/lightstep.md [hunk 1]
```diff
@@ -8,9 +8,20 @@ Examples of configuring Tracetest with Lightstep can be found in the [`examples`
 
 ## Configuring OpenTelemetry Collector to Send Traces to both Lightstep and Tracetest
 
-In your OpenTelemetry Collector config file, make sure to set the `exporter` to `otlp/tt`, with the `endpoint` pointing to your Tracetest instance on port `21321`. If you are running Tracetest with Docker, the endpoint might look like this `http://tracetest:21321`.
+In your OpenTelemetry Collector config file:
```
Comment (danielbdias) [inline docs/docs/configuration/connecting-to-data-stores/lightstep.md:11]: We will need to do this on our [current Datadog documentation](https://github.com/kubeshop/tracetest/blob/main/docs/docs/configuration/connecting-to-data-stores/datadog.md) too. I can do that on my PR which is adding more examples and docs to Datadog.

**Review by danielbdias — APPROVED** [review]

File: docs/docs/configuration/connecting-to-data-stores/lightstep.md (hunk 1 above)
Comment (adnanrahic) [reply docs/docs/configuration/connecting-to-data-stores/lightstep.md:11]: Makes sense. I will make sure to check where updates are needed once I merge this one.

Comment (adnanrahic) [reply docs/docs/configuration/connecting-to-data-stores/lightstep.md:11]: @danielbdias I updated the DD config as well in this PR.

**Pull request merged**